openpower/package/openpower-pnor/Config.in

```diff
@@ -154,6 +154,18 @@ config BR2_OPENPOWER_TARGETING_ECC_FILENAME
         help
             String used to define name of openpower targeting binary file, ecc protected
 
+config BR2_OCMBFW_FILENAME
+        string "Name of the OCMBFW original file"
+        default "ocmbfw.bin"
+        help
+            String used to define name of OBMC original file (before any processing occurs)
+
+config BR2_OCMBFW_PROCESSED_FILENAME
```

Comment:
Well, I guess this is where the convention of having a processed name is.  It makes more sense why you tried to keep the processed filename in the pnor .pl scripts.